src/components/src/AppHeaderLogin/appheaderlogin.main.tsx

```diff
@@ -352,23 +384,56 @@ class AppHeaderLoginMain extends Component<AppHeaderLoginMainProps, AppHeaderLog
               </button>
               <div data-region="authMainRegion" className="auth-nav-container dropdown-menu dropdown-menu-right" aria-label="header_navbar_login_button ">
                 <ul data-el-container="global.profileMenu" className="auth-profile-menu-list">
-                  {userName !== 'undefined' ? (
-                    <li className="dropdown-header">
-                      {userName}
+                  {Config.b2b.enable ? (
+                    <li className="dropdown-item shop-for">
+                      <span className="user-name">{b2bUserName}</span>
```

Comment:
The value for this is displaying "lastname firstname" in the incorrect order
![image](https://user-images.githubusercontent.com/39800563/77658341-f1042600-6f4c-11ea-8b06-71f2caa1fe52.png)